src/lints/trait_default_impl_removed.ron

```diff
@@ -0,0 +1,62 @@
+SemverQuery(
+    id: "trait_default_impl_removed",
+    human_readable_name: "pub trait default method impl removed",
+    description: "A non-sealed public trait default method impl was removed",
+    required_update: Major,
+    lint_level: Deny,
+    reference_link: Some("https://doc.rust-lang.org/book/ch10-02-traits.html#default-implementations"),
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        sealed @filter(op: "!=", value: ["$true"])
```

Comment:
Right now, none of the test cases include sealed traits so this is never tested. We could remove this clause and the corresponding clause in the `current` arm of the query, and all the tests would still pass. That's a bit scary!

Would you mind adding some test cases that ensure this lint only triggers when the trait wasn't sealed to start with?

Reply:
Added the fully sealed case to test that case